### ISSUE TITLE
Default Menu State: Open

### ISSUE THREAD

**Issue by mattttt** [raintank/grafana#54]
For the raintank build, I'd like to side menu to be open for new users. 

Unsure how state is set/saved (if it is), so this might need some quick discussion with @awoods or @torkelo

**Comment by nopzor1200** [comment]
Probably a good thing for brand new Grafana users too (and then keep hiding it once they close it)?

**Comment by mattttt** [comment]
Agreed.

**Comment by bulletfactory** [comment]
I like his idea too.

**Comment by torkelo** [comment]
It does not make much sense for the state to be open by default for Grafana I think as you seldom need the sidemenu. The sidemenu state is remembered via local storage (browser storage), so the state is remembered when doing full page load.

**Comment by nopzor1200** [comment]
I agree it shouldn't be open by default. I also agree you seldom need the side menu (especially as the side menu stands now).

I'm not sure if this was @mattttt original intent (I think it was) or what @bulletfactory thought, but let me rephrase what I thought to be the request:

"On first ever login, the side menu bar should be open"

From there, once they closed it for the first time it'd never appear again (even if they logged out and logged back in), unless they opened it up explicitly 

It's a one time thing per user the way I imagine it. Not a default. The intent would be to educate the user that it even exists, how to open/close it, and provide a more 'cosy' introduction to their first 5 seconds in Grafana. (eg. help them add datasource, users, which are the first thing new users need to succeed at doing)

And I could be totally butchering @mattttt suggestion into something else, and maybe he did mean default.

And you could still think it's a bad idea for mainline grafana :)

**Comment by torkelo** [comment]
@nopzor1200 no, I agree, that is something I have also felt would be good (that it is open on first login, or first 2 logins)

**Comment by woodsaj** [comment]
this looks pretty easy to implement.

just need to change this line to the below.
https://github.com/grafana/grafana/blob/master/public/app/services/contextSrv.js#L48
```
    this.sidemenu = store.getBool('grafana.sidemenu', true);
```

This will set the sidemenu to open if the the users local browser storage does not yet have any value stored.

**Comment by nopzor1200** [comment]
It's not quite the ideal scenario but it's an improvement (It doesn't actually allow for only showing the sidemenu for the first 1 or 2 logins during the lifetime of the user. It just changes the default state)

Let's make this change for raintank/grafana for now, especially since I think we'll be using the sidebar for more stuff imminently..

**Comment by woodsaj** [comment]
submitted PR upstream for this.

https://github.com/grafana/grafana/pull/1923

**Comment by woodsaj** [comment]
commits merged upstream.
